lib/language_pack/ruby.rb

```diff
@@ -393,6 +403,11 @@ def setup_language_pack_environment(ruby_layer_path:, gem_layer_path:)
       paths << ENV["PATH"]
 
       ENV["PATH"] = paths.join(":")
+
+      ENV["BUNDLE_WITHOUT"] = env("BUNDLE_WITHOUT") || bundle_default_without
+      ENV["BUNDLE_PATH"] = bundle_path
+      ENV["BUNDLE_BIN"] = bundler_binstubs_path
+      ENV["BUNDLE_DEPLOYMENT"] = "1"
```

Comment:
If we display all the env vars when running the command, do we want to make `BUNDLE_DEPLOYMENT` configurable?

Reply:
I don't think there's a reason to deploy without `--deployment` mode unless they're using a windows Gemfile.lock.